mock-core-configs/mock-core-configs.spec

```diff
@@ -68,9 +74,28 @@ case $HOST in
     for config in etc/mock/fedora-*-*.cfg; do
         version=$(echo "$config" | cut -d- -f2)
         if test $version = rawhide || test $version -ge 31; then
-            echo "config_opts['use_bootstrap_image'] = True" >> "$config"
+            bootstrap_image_ON "$config"
         fi
     done
+
+    for config in etc/mock/*cfg; do
+        case $config in
+            *alma+epel-9*) ;&
+            *almalinux-9*) ;&
+            *centos-stream-9*) ;&
+            *centos-stream+epel-9*) ;&
+            *centos-stream+epel-next-9*) ;&
+            *eurolinux-9*) ;&
+            *oraclelinux-9*) ;&
+            *oraclelinux+epel-9*) ;&
+            *rhel-9*) ;&
+            *rhel+epel-9*) ;&
+            *rocky-9*) ;&
+            *rocky+epel-9*) ;&
+            FIX_IMAGE)
```

Comment:
Why this line?

Reply:
I'll clean this up. I wanted to have the lines alphabetically ordered and prepared for updates in the future.